src/main/java/minijava/lexer/Token.java

```diff
@@ -0,0 +1,59 @@
+package minijava.lexer;
+
+/** Lexer token */
+public class Token {
+
+  private final Terminal terminal;
+  private final Location location;
+  private final int content;
+  private final StringTable stringTable;
+
+  public Token(Terminal terminal, Location location, int content, StringTable stringTable) {
```

Comment:
Hmm... I don't think that's a good idea, the test suite will also have to create tokens.